forward-message-sendgrid/README.md

```diff
@@ -1,3 +1,9 @@
+
+### This Repo is now out of date. 
```

Comment:
As replied here https://github.com/twilio-labs/function-templates/pull/501
If the code still works, I do not see that this would be out of date.